packages/venia-concept/src/components/Checkout/paymentsForm.js

```diff
@@ -21,8 +21,26 @@ const DEFAULT_FORM_VALUES = {
     addresses_same: true
 };
 
-const BillingAddressFields = ({ classes, countries }) => {
-    return (
+const PaymentsFormChildren = props => {
```

Comment:
`PaymentsFormChildren` should also be extracted into its own file, no?

Also, consider `PaymentsFormItems`, if you'd like to follow the pattern we have in a couple other places.

Reply:
I wasn't sure about this -- the component wrapping it is literally just one layer:

```
return (
  <div>
    <PaymentsFormChildren {...props} />
   </div>
)
```
We can probably just get rid of this separated component.

Reply:
Oh I see -- it's pretty involved. Maybe I need to break out the `BillingAddressFields` too...